File: src/main/java/no/nav/syfo/domain/Kommentar.java
package no.nav.syfo.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class Kommentar {
    public Long id;
    public long tiltakId;
    public String tekst;
    public String sistEndretAvAktoerId;
    public LocalDateTime sistEndretDato;
    public String opprettetAvAktoerId;
    public LocalDateTime opprettetDato;
}

File: src/main/kotlin/no/nav/syfo/api/v2/domain/oppfolgingsplan/Kommentar.kt
package no.nav.syfo.api.v2.domain.oppfolgingsplan

import java.time.LocalDateTime

data class Kommentar(
    val id: Long,
    val tekst: String?,
    val opprettetTidspunkt: LocalDateTime,
    val sistEndretDato: LocalDateTime,
    val opprettetAv: Person,
    val sistEndretAv: Person
)
